src/test/tv/codely/scala_http_api/module/user/UserUnitTestCase.scala

```diff
@@ -0,0 +1,13 @@
+package tv.codely.scala_http_api.module.user
+
+import tv.codely.scala_http_api.module.UnitTestCase
+import tv.codely.scala_http_api.module.user.domain.{User, UserRepository}
+
+protected[user] trait UserUnitTestCase extends UnitTestCase {
```

Comment:
Which naming convention we'll follow for the unit tests? "unit"? "logic"?

Reply:
I think `unit` is more common 😬